deploy/roles/mesos_master/tasks/main.yml

```diff
@@ -548,3 +548,23 @@
   tags:
     - mesos
     - mesos-master
+
+- name: Set recursive admin permissions on log folders
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    recursive: yes
+    state: present
+  with_items: "{{ mesos_master_log_file_locations }}"
+
+- name: Set default admin permissions on log folders
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    default: yes
+    state: present
+  with_items: "{{ mesos_master_log_file_locations }}"
```

Comment:
@stirlingv We also need to add these permissions tasks to the mesos_agent role, but only for /var/log/mesos and not /var/log/zookeeper.  Thanks!